recipes/default.rb

```diff
@@ -4,81 +4,54 @@
 #
 # Copyright (c) 2016 The Authors, All Rights Reserved.
 
-# install wget
-package 'wget' do
-  action :install
-end
-
-# install epel-release
-package 'epel-release' do
-  action :install
-end
+include_recipe 'delugeserver::yum_setup'
 
 # create system directories
 directory '/.downloads' do
   owner 'root'
   group 'root'
 end
 
-# download nux-dextop rpm
-execute 'nux-dextop_download' do
-  not_if do ::File.exists?('/.downloads/nux-dextop-release-0-5.el7.nux.noarch.rpm') end
-  command 'wget http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm -P /.downloads/'
-  action :run
-end
-
-# install nux-dextop
-package 'nux-dextop' do
-  source '/.downloads/nux-dextop-release-0-5.el7.nux.noarch.rpm'
-  action :install
-end
-
 # install deluge-daemon
-package 'deluge-daemon' do
+package 'deluge' do
```

Comment:
one concern is making sure that the deluged service is started and stop as early as possible so that we give it time enough to generate the directories and core config files before we start to mess with them later on in the recipe